tsc: Cannot find name 'GPUTexture'

When building with typescript on a project, I'm receiving the following error:

```
$ tsc
node_modules/@types/three/src/textures/ExternalTexture.d.ts:4:35 - error TS2552: Cannot find name 'GPUTexture'. Did you mean 'Texture'?

4     sourceTexture: WebGLTexture | GPUTexture | null;
                                    ~~~~~~~~~~

node_modules/@types/three/src/textures/ExternalTexture.d.ts:8:48 - error TS2552: Cannot find name 'GPUTexture'. Did you mean 'Texture'?

8     constructor(sourceTexture?: WebGLTexture | GPUTexture | null);
                                                 ~~~~~~~~~~


Found 2 errors in the same file, starting at: node_modules/@types/three/src/textures/ExternalTexture.d.ts:4
```

This appears to be recently introduced in https://github.com/three-types/three-ts-types/pull/1791

Dependencies:

```json
  "dependencies": {
    "@codemirror/autocomplete": "^6.18.7",
    "@codemirror/commands": "^6.8.1",
    "@codemirror/lang-yaml": "^6.1.2",
    "@codemirror/search": "^6.5.11",
    "@codemirror/state": "^6.5.2",
    "@codemirror/view": "^6.38.2",
    "@fsegurai/codemirror-theme-bundle": "^6.2.2",
    "@rollup/plugin-node-resolve": "^16.0.1",
    "@rollup/plugin-terser": "^0.4.4",
    "@types/three": "^0.180.0",
    "@webgpu/types": "^0.1.65",
    "codemirror": "^6.0.2",
    "rollup": "^4.50.2",
    "three": "^0.180.0",
    "typescript": "^5.9.2"
  }
```

Thanks in advance! :-)

--- a/types/three/src/textures/ExternalTexture.d.ts
+++ b/types/three/src/textures/ExternalTexture.d.ts
@@ -1,3 +1,5 @@
+/// <reference types="@webgpu/types" />
+
 import { Texture } from "./Texture.js";
 
 declare class ExternalTexture extends Texture {

